### ISSUE TITLE
Encourage users to connect over HTTPS / Tor

### ISSUE THREAD

**Issue by cstenglein** [cstenglein/raspiblitz-web#209]
We should encourage users not to access the WebUI over HTTP, but HTTPS / Tor.

**Comment by rootzoll** [comment]
Yeah - thats a longer discussion and might get improved over time. For the beginning we might just do the HTTP in the local network like the other node projects do by default and HTTPS als alternative for those that can handle it the accept self-signed cert warnings.

Tor would make the Node publicly available - which is especially for during setup with default password maybe not a good practice. Later if we have the LCD display part done/refactored, maybe a special code/password on the display can help, etc. but I would not rush that.

Also with Tor the user might still need to open the normal browser first ... because Tor-Browser cannot open local network sides like "http://192.168.178.61" and then would need to copy the hiddenservice-address from there to the tor browser, etc. All quite complicated for Beginners. So thats something for a v1.8.1 to find a sweetspot and improve here.
